core/org/commcare/core/parse/UserXmlParser.java

```diff
@@ -82,10 +78,16 @@ public User parse() throws InvalidStructureException, IOException, XmlPullParser
             }
         }
 
+        addCustomData(u);
+
         commit(u);
         return u;
     }
 
+    public void addCustomData(User u){
+        //Don't do anything in base class
```

Comment:
This doesn't quite sound right to me, don't we use custom user data in both j2me and android?